scripts/urlgrabber

```diff
@@ -178,8 +178,8 @@ class client_options:
         try:
             optlist, args = getopt.getopt(sys.argv[1:], short_options,
                                           long_options + ug_long)
-        except getopt.GetoptError, e:
-            print >>sys.stderr, "Error:", e
+        except getopt.GetoptError as e:
+            print("Error:", e, file=sys.stderr)
```

Comment:
Please ensure everything using `print()` has the `__future__` import so that it disables the legacy behavior.

```
from __future__ import print_function
```